var/spack/repos/builtin/packages/py-distributed/package.py

```diff
@@ -24,7 +24,7 @@ class PyDistributed(PythonPackage):
     depends_on('py-cloudpickle@0.2.2:', type=('build', 'run'), when='@:2.16.0')
     depends_on('py-cloudpickle@1.3.0:', type=('build', 'run'), when='@2.17.0:2.20.0')
     depends_on('py-cloudpickle@1.5.0:', type=('build', 'run'), when='@2.21.0:')
-    depends_on('py-contextvars', type=('build', 'run'), when='^python@:3.6')
+    depends_on('py-contextvars', type=('build', 'run'), when='@2: ^python@:3.6')
```

Comment:
I don't see this requirement in 2.10.0, it might only be needed in 2020.12.0

Reply:
good point, fixed